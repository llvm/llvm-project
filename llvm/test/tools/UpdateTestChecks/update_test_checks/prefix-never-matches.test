# RUN: cp -f %S/Inputs/prefix-never-matches.ll %t.ll
# RUN: %update_test_checks %t.ll
# RUN: diff -u %t.ll %S/Inputs/prefix-never-matches.ll.expected