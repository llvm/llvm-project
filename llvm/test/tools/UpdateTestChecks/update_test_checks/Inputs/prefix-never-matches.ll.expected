; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
; RUN: opt -O0 -S < %s  | FileCheck %s -check-prefix=A
; RUN: opt -O3 -S < %s  | FileCheck %s -check-prefix=A

define i32 @foo(i32 %i) {
  %r = add i32 1, 1
  ret i32 %r
}
;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
; A: {{.*}}
